eng/format-job.yml

```diff
@@ -14,12 +14,12 @@ jobs:
     buildConfig: ${{ parameters.buildConfig }}
     archType: ${{ parameters.archType }}
     osGroup: ${{ parameters.osGroup }}
-    osIdentifier: ${{ parameters.osIdentifier }}
+    osSubgroup: ${{ parameters.osSubgroup }}
     container: ${{ parameters.container }}
     crossrootfsDir: ${{ parameters.crossrootfsDir }}
     timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
-    name: ${{ format('format_{0}_{1}', parameters.osIdentifier, parameters.archType) }}
-    displayName: ${{ format('Formatting {0} {1}', parameters.osIdentifier, parameters.archType) }}
+    name: ${{ format('format_{0}{1}_{2}', parameters.osGroup, parameters.osSubgroup, parameters.archType) }}
```

Comment:
So if `parameter.osSubgroup` can take values from *empty* or `_rhel6` or `_musl` then we don't need preceding underscore in the format string?

Reply:
Hmm, I'm not sure if I understand your question. The {0}{1} section in 'name' corresponds to the plain concatenation of osGroup and osSubgroup i.e. the equivalent of previous osIdentifier. The other underscores in the format string are the pre-existing ones, my change shouldn't be actually renaming any jobs. Could you please clarify what you had in mind?

Reply:
Please dismiss - I was looking at the wrong placeholder.